also a revert of
https://git.kernel.org/pub/scm/docs/man-pages/man-pages.git/commit/share/mk/install/man.mk?id=30c38a8bf8ae8f4a6e71d3b8ac4abf0a40778f1e
because bsd install does only a copy and does not allow to 'install' /dev/stdin
via reading from it
--
--- a/share/mk/install/html.mk
+++ b/share/mk/install/html.mk
@@ -23,7 +23,7 @@ _htmlpages_rm   := $(addsuffix -rm,$(wildcard $(_htmlpages)))
 
 $(_htmlpages): $(_htmldir)/%: $(_HTMLDIR)/% | $$(@D)/
 	$(info INSTALL	$@)
-	$(INSTALL_DATA) -T $< $@
+	$(INSTALL_DATA) $< $@
 
 
 .PHONY: install-html
--- a/share/mk/install/man.mk
+++ b/share/mk/install/man.mk
@@ -60,10 +60,10 @@
 
 $(_manpages):
 	$(info INSTALL	$@)
-	<$< \
+	$(INSTALL_DATA) $< $@
 	$(SED) $(foreach s, $(MANSECTIONS), \
-		-e '/^\.so /s, man$(s)/\(.*\)\.$(s)$$, $(notdir $(man$(s)dir))/\1$(man$(s)ext)$(Z),') \
-	| $(INSTALL_DATA) -T /dev/stdin $@
+		-i '' -e '/^\.so /s, man$(s)/\(.*\)\.$(s)$$, $(notdir $(man$(s)dir))/\1$(man$(s)ext)$(Z),') \
+		$@
 ifeq ($(LINK_PAGES),symlink)
 	if $(GREP) '^\.so ' <$@ >/dev/null; then \
 		$(GREP) '^\.so ' <$@ \
